src/templates/storage-view-edit.html

```
@@ -10,7 +11,7 @@ <h3 title='{{ 'Toggle visibility'|trans }}' data-action='toggle-next' data-opene
             <li class='list-group-item countable'>
               <div class='d-flex flex align-items-center'>
                 <div>
-                  <span class='mr-2'><span class='malleableColumn hl-hover p-1 rounded' data-ma-type='float' data-endpoint='{{ Entity.entityType.value }}/{{ Entity.id }}/containers' data-id='{{ container.id }}' data-target='qty_stored'>{{ container.qty_stored }}</span></span> <span class='mr-2'><span class='malleableQtyUnit hl-hover p-1 rounded' data-endpoint='{{ Entity.entityType.value }}/{{ Entity.id }}/containers' data-id='{{ container.id }}'>{{ container.qty_unit }}</span> {{ 'in'|trans }} {{ container.full_path }}</span>
+                  <span class='mr-2'><span class='malleableColumn hl-hover p-1 rounded' data-ma-type='float' data-endpoint='{{ Entity.entityType.value }}/{{ Entity.id }}/containers' data-id='{{ container.id }}' data-target='qty_stored'>{{ container.qty_stored }}</span></span> <span class='mr-2'><span class='malleableQtyUnit hl-hover p-1 rounded' data-endpoint='{{ Entity.entityType.value }}/{{ Entity.id }}/containers' data-id='{{ container.id }}'>{{ container.qty_unit }}</span> {{ 'in'|trans }} {% if Entity.isReadOnly %}{{ container.full_path }}{% else %}<span role='button' tabindex='0' class='hl-hover p-1 rounded' data-action='move-container' data-id='{{ container.id }}' data-current-storage-id='{{ container.storage_id }}' title='{{ 'Move to another storage location'|trans }}'>{{ container.full_path }}</span>{% endif %}</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)